### PR TITLE
add: `helix`

### PR DESCRIPTION
**Pull request by tranzystorek-io**

### CODE REVIEW

File: packages/helix/helix.pacscript [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+build() {
+    cargo build -j$(nproc) --release --locked
+}
+
+install() {
+    sudo install -Dm755 target/release/hx -t "${pkgdir}/usr/lib/${name}"
+
+    wget -q "https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/${name}/hx"
+    sudo install -Dm755 hx -t "${pkgdir}/usr/bin"
```
Comment (Henryws) [inline packages/helix/helix.pacscript:22]: Couldn't this be replaced with a simple `echo "script"`?

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/helix/helix.pacscript (hunk 1 above)
Comment (tranzystorek-io) [reply packages/helix/helix.pacscript:22]: Sure, let me adjust that in a sec 😄

Comment (tranzystorek-io) [reply packages/helix/helix.pacscript:22]: Do you prefer me squashing this adjustment or just adding as another commit?

Comment (Henryws) [reply packages/helix/helix.pacscript:22]: Whatever works for you

File: packages/helix/helix.pacscript [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+build() {
+    cargo build -j$(nproc) --release --locked
+}
+
+install() {
+    sudo install -Dm755 target/release/hx -t "${pkgdir}/usr/lib/${name}"
+
+    echo '#!/bin/sh\nHELIX_RUNTIME=/usr/lib/helix/runtime exec /usr/lib/helix/hx "$@"' > hx
```
Comment (Henryws) [inline packages/helix/helix.pacscript]: I don't think newlines work on single quotes

File: packages/helix/helix.pacscript (hunk 1 above)
Comment (tranzystorek-io) [reply packages/helix/helix.pacscript]: Done

File: packages/helix/helix.pacscript [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+description="A post-modern modal text editor"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
+hash="c9b184d53bab51e7a9fe81dc8c191a853177abba67ea408e968937f31b50b45a"
+
+prepare() {
+    true
+}
+
+build() {
+    cargo build -j$(nproc) --release --locked
```
Comment (Henryws) [inline packages/helix/helix.pacscript:15]:
```suggestion
    cargo build -j"$(nproc)" --release --locked
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/helix/helix.pacscript (hunk 2 above)
Comment (tranzystorek-io) [reply packages/helix/helix.pacscript]: The file ends up correct when I tested the package

File: packages/helix/helix.pacscript [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+url="https://github.com/helix-editor/helix/archive/refs/tags/${version}.tar.gz"
+build_depends="cargo git"
+description="A post-modern modal text editor"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
+hash="c9b184d53bab51e7a9fe81dc8c191a853177abba67ea408e968937f31b50b45a"
+
+prepare() {
+    true
+}
+
```
Comment (Henryws) [inline packages/helix/helix.pacscript:13]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
Fails on Debian.

File: packages/helix/helix.pacscript (hunk 3 above)
Comment (tranzystorek-io) [reply packages/helix/helix.pacscript]: Done

File: packages/helix/helix.pacscript (hunk 4 above)
Comment (tranzystorek-io) [reply packages/helix/helix.pacscript]: Done

**Comment by tranzystorek-io** [comment]
Looking at the [named-profiles] feature, helix needs at least Rust 1.57, so Debian testing+

**Review by Henryws — APPROVED** [review]

**Pull request merged**